test/back_end/api_student_survey_test.spec.js

```diff
@@ -0,0 +1,39 @@
+// This is the mocha testing file for get_info.php
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+chai.use(chaiHttp);
+const expect = chai.expect;
+const base64o = require("../../src/base64js.min.js");
+const base64 = base64o.base64;
+
+describe("Test Student Survey API", function() {
+    it("Should return TA info for ta11", async function() {
+        try {
+            let fetched = await fetch(
+                "http://localhost:3000/student_survey.php?what=get_ta&user_id=student1&override_token=1KSOSIWXGJ6R2L93AEG7"
+            );
+            expect(fetched).to.have.status(200);
+            let fetchedJSON = await fetched.json();
+            expect(fetchedJSON).to.have.nested.property("TYPE");
+            expect(fetchedJSON).to.have.nested.property("DATA");
+            expect(fetchedJSON.TYPE).to.equal("ta_package");
+        } catch (error) {
+            throw error;
```

Comment:
Since we're just throwing the error again, there's no need for the `try` statement.